ui/index.d.ts

```diff
@@ -133,8 +133,9 @@ export interface IChildProvider {
     /**
      * Implement this if you want the 'create' option to show up in the node picker
      * @param options User-defined options that are passed to the IAzureParentTreeItem.createChild call
+     * @param resourceGroup The name of the resource group the app will be created under programmatically
      */
-    createChild?(node: IAzureNode, showCreatingNode: (label: string) => void, userOptions?: any): Promise<IAzureTreeItem>;
+    createChild?(node: IAzureNode, showCreatingNode: (label: string) => void, userOptions?: any, resourceGroup?: string): Promise<IAzureTreeItem>;
```

Comment:
You shouldn't be changing anything in the ui package for this. `resourceGroup` should be a _part_ of userOptions (that's the whole point of it being type `any`)